local.go

```diff
@@ -328,32 +334,9 @@ func evaluate(funcs functions, funcInfo *FunctionInfo, evalType EvaluationType,
 }
 
 // update updates the stats given the new evaluation
```

Comment:
It seems like at this point, update can just go away. I had originally wanted to separate with a future Global, but it'll probably need it's own function anyway.